src/google/protobuf/compiler/csharp/csharp_options.h

```diff
@@ -0,0 +1,60 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
The cpp options are under `libprotoc_la_SOURCES` - I think that's probably appropriate for the C# ones too.